<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="Joes-MacBook-Pro.local" name="testSuiteA.IdeTest" tests="12" failures="0" timestamp="29 Dec 2013 20:37:11 GMT" time="174.190" errors="1">
  <testcase name="changeFWTest" time="16.584" classname="testSuiteA.IdeTest"/>
  <testcase name="rmWebhookTest" time="8.512" classname="testSuiteA.IdeTest"/>
  <testcase name="addWebhookTest" time="14.589" classname="testSuiteA.IdeTest"/>
  <testcase name="searchTest" time="34.489" classname="testSuiteA.IdeTest"/>
  <testcase name="rmBondTest" time="12.414" classname="testSuiteA.IdeTest"/>
  <testcase name="deleteFImpTest" time="6.563" classname="testSuiteA.IdeTest">
    <error type="java.lang.AssertionError" message="expected:&lt;true&gt; but was:&lt;false&gt;">
      <![CDATA[java.lang.AssertionError: expected:<true> but was:<false>
	at org.testng.Assert.fail(Assert.java:89)
	at org.testng.Assert.failNotEquals(Assert.java:489)
	at org.testng.Assert.assertTrue(Assert.java:37)
	at org.testng.Assert.assertTrue(Assert.java:47)
	at testSuiteA.IdeTest.deleteFImpTest(IdeTest.java:370)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:39)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:25)
	at java.lang.reflect.Method.invoke(Method.java:597)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:80)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:673)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:842)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1166)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
	at org.testng.TestRunner.runWorkers(TestRunner.java:1178)
	at org.testng.TestRunner.privateRun(TestRunner.java:757)
	at org.testng.TestRunner.run(TestRunner.java:608)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:334)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:329)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:291)
	at org.testng.SuiteRunner.run(SuiteRunner.java:240)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1158)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1154)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1083)
	at org.testng.TestNG.run(TestNG.java:999)
	at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:217)
	at org.apache.maven.surefire.testng.TestNGXmlTestSuite.execute(TestNGXmlTestSuite.java:84)
	at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:92)
	at org.apache.maven.surefire.booter.ForkedBooter.invokeProviderInSameClassLoader(ForkedBooter.java:200)
	at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:153)
	at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:103)
]]>
    </error>
  </testcase>
  <testcase name="sidebarTest" time="28.894" classname="testSuiteA.IdeTest"/>
  <testcase name="loginTest" time="4.436" classname="testSuiteA.IdeTest"/>
  <testcase name="opsDeployTest" time="18.772" classname="testSuiteA.IdeTest"/>
  <testcase name="addFImpTest" time="5.460" classname="testSuiteA.IdeTest"/>
  <testcase name="assignDevTest" time="12.422" classname="testSuiteA.IdeTest"/>
  <testcase name="deleteDevTest" time="11.055" classname="testSuiteA.IdeTest"/>
</testsuite>
